source/MultiGrid/MultiGrid.js

```diff
@@ -72,23 +72,28 @@ export default class MultiGrid extends Component {
     rowIndex = 0
   } = {}) {
     const { fixedColumnCount, fixedRowCount } = this.props
-
-    this._bottomLeftGrid && this._bottomLeftGrid.measureAllCells({
-      columnIndex,
-      rowIndex: rowIndex - fixedRowCount
+    const col = (columnIndex >= fixedColumnCount ? columnIndex - fixedColumnCount : columnIndex)
+    const row = (rowIndex >= fixedRowCount ? rowIndex - fixedRowCount : rowIndex)
```

Comment:
Instead of dealing with it in MultiGrid, perhaps a change to [CellSizeAndPositionManager#resetCell](https://github.com/bvaughn/react-virtualized/blob/080572f2e74af72416ba2f542ab6a6ea6aa3476a/source/Grid/utils/CellSizeAndPositionManager.js#L180) instead?

```
resetCell (index: number): void {
    let guarded = Math.max(index, 0)
    this._lastMeasuredIndex = Math.min(this._lastMeasuredIndex, guarded - 1)
  }
```